### DISCUSSION TITLE
编辑器删除素材，页面loading

### DISCUSSION THREAD

**Discussion by lixx3000** [galacean/discussion#83] [Editor]
![image](https://github.com/galacean/discussion/assets/45725941/7527d7e0-c145-4d60-ae59-af13a97b930e)

**Comment by eyworldwide** [comment]
@lixinhang 页面 loading 具体表现是怎么样的？另外，是删除所有素材都会出现这种情况还是个别素材？

**Comment by lixx3000** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by lixx3000** [comment]
https://galacean.antgroup.com/editor#/project/12100001
这是我复现的地址
我删除的是这个文件夹下的素材
![image](https://github.com/galacean/discussion/assets/45725941/137138b6-9e90-4142-a325-e90aa35334e4)

**Comment by eyworldwide** [comment]
@lixx3000 **Internal** 下的资产是内置资产，理论上是只读的，不可以删除是合理的，但可以写入也不对，我判断是个 bug。

**Comment by luzhuang** [comment]
已修复

**Comment by lixx3000** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
